xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -67,9 +69,9 @@ CVideoPlayerVideo::CVideoPlayerVideo(CDVDClock* pClock
   m_iDroppedRequest = 0;
   m_fForcedAspectRatio = 0;
 
-  // 128 MB allows max bitrate of 128 Mbit/s (e.g. UHD Blu-Ray) during 8 seconds
+  // 128 MB allows max bitrate of 128 Mbit/s (e.g. UHD Blu-Ray) during m_messageQueueTimeSize seconds
   m_messageQueue.SetMaxDataSize(128 * 1024 * 1024);
```

Comment:
Same: keep code comment or change 128 constant proportionally to m_messageQueueTimeSize

e.g. if queue time is 4.0s max data size is 64 * 1024 * 2024